mne/bem.py

```diff
@@ -1823,14 +1823,9 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
         out_fname = op.join(flash_bem_dir, surf + '.tri')
         shutil.move(op.join(bem_dir, surf + '.tri'), out_fname)
         nodes, tris = read_tri(out_fname, swap=True)
-        vol_info = _extract_volume_info(flash5_reg)
-        if vol_info is None:
-            warn('nibabel is required to update the volume info. Volume info '
-                 'omitted from the written surface.')
-        else:
-            vol_info['head'] = np.array([20])
-        write_surface(op.splitext(out_fname)[0] + '.surf', nodes, tris,
-                      volume_info=vol_info)
+        # Do not write volume info here because the tris are already in
+        # standard Freesurfer coords
```

Comment:
how did this ever work including with forward computation???

Reply:
It's likely that the C version does not write this metadata, so never introduced the problem (?). In any case, when we read the surface, we don't apply and transform based on the metadata. But `freeview` does.

Reply:
I guess it's not restricted to weird orientations actually. On `master` even with the `sample` data running `mne flash_bem --noconvert -s sample -d ~/mne_data/MNE-sample-data` then `mne freeview_bem_surfaces -s sample -d ~/mne_data/MNE-sample-data/subjects/` gives:

<img width="1325" alt="Screen Shot 2019-03-19 at 21 42 56" src="https://user-images.githubusercontent.com/2365790/54600069-13d54200-4a90-11e9-90b6-89185f98ba77.png">

But `plot_alignment` is fine:

<img width="912" alt="Screen Shot 2019-03-19 at 21 41 57" src="https://user-images.githubusercontent.com/2365790/54600053-0a4bda00-4a90-11e9-9f2d-8fac188635dc.png">

And on this PR Freeview is fixed:

<img width="1325" alt="Screen Shot 2019-03-19 at 21 44 58" src="https://user-images.githubusercontent.com/2365790/54600165-48e19480-4a90-11e9-877c-e448614babe6.png">

And the `plot_alignment` is unchanged (data not shown).